opendatahub-io/odh-dashboard
# Fixing frontend packages with vulnerabilities

## Description
Upgraded dependencies/sub-dependencies to resolve vulnerabilities.
This will resolve vulnerabilities issues raised on [Synk app](https://app.snyk.io/org/red-hat-openshift-data-science-rhods/project/1d87351a-04b6-4d75-b13b-9187e9f55bc5)

## How Has This Been Tested?
* Pull this branch locally and `npm i`
* Run `cd frontend/ && npm audit`
* You should see `found 0 vulnerabilities`
* Try running odh-dashboard

## Test Impact
No test, just upgrading packages.

## Request review criteria:



Self checklist (all need to be checked):

If you have UI changes: 


After the PR is posted & before it merges:
